### PR TITLE
fix: Voice settings flaws fixed

### LINKED ISSUE #1956: Disabled settings options seem to be working
**Describe the bug**
In Settings fragment some options work only if user is logged in.
But the UI controls work.

**Expected Behaviour**
User should not be allowed to change options in UI which are disabled in code

**Steps to reproduce it**
1.Open Settings fragment without logging in
2.Try to change speech options

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |[ Moto G4 Plus] |
|Android Version| [ Nougat 7.0] |

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #1956 

Changes: 
The settings pretended to be deactivated is user is not logged in, but the code was not written properly. So I have changed the code.

Screenshots for the change: 
Screenshot 2019-05-15 at 5 53 43 PM

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt [hunk 1]
```diff
@@ -112,9 +112,13 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
         if (!utilModel.isLoggedIn()) {
             displayEmail.title = "Not logged in"
             displayEmail.isEnabled = true
+            speechOutput.isEnabled = false
+            speechAlways.isEnabled = false
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: Why?

Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: sir, this was the flaw that was stated in the issue. User should be allowed to change voice setting only if he is logged in. The code base written below also showed similar approach.

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: Why should he only be allowed when he's logged in?

Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: Sir, the issue created by @ravi-kishan stated that thing only. Also, when I analyzed the code base. It pretended to do the same. It had used the anonymity function to check wether the user is signed in or not.

**Comment by iamareebjamal** [comment]
Show something to the user that to change this setting, you need to be logged in

**Comment by atm1504** [comment]
> I think this needs to be discussed why should we do this

Shall I put this topic in the gitter channel?

**Comment by iamareebjamal** [comment]
Move to different issue for now

**Comment by atm1504** [comment]
> Move to different issue for now

Okk Sir. I am trying to solve the issues that are present till I get the equipment for the smart speaker to work with.

**Comment by mariobehling** [comment]
The linked issue is closed already. Therefore closing this PR.

**Pull request closed without merging**